caluma/caluma_form/models.py

```diff
@@ -208,8 +208,44 @@ class Document(UUIDModel):
     form = models.ForeignKey(
         "caluma_form.Form", on_delete=models.DO_NOTHING, related_name="documents"
     )
+    source = models.ForeignKey(
+        "self",
+        blank=True,
+        null=True,
+        help_text="Reference this document has been copied from",
+        related_name="+",
```

Comment:
Up for discussion: Should there be a way to query copies of a document? `related_name="copies"`?

Reply:
Very good question @winged @czosel

Reply:
I could imagine that we’re gonna need this somewhen, yeah. If there are no downsides I’d say let’s add it already now.

Reply:
I don't see any downsides. Of course, there is a bit more needed than setting the `related_name` in order to make it accessible with graphql.

Reply:
Right - I’d wait with extending the API though until we have an actual use-case.

Reply:
Added `related_name="copies"`